buildroot-external/kernel/docker.config

```diff
@@ -4,6 +4,8 @@ CONFIG_CFS_BANDWIDTH=y
 CONFIG_FAIR_GROUP_SCHED=y
 CONFIG_NET_SCHED=y
 
+CONFIG_RT_GROUP_SCHED=n
+
```

Comment:
That doesn't work with Kconfig. `# CONFIG_RT_GROUP_SCHED is not set` should though.

Reply:
good point :D